kumascript/macros/Link.ejs

```diff
@@ -5,7 +5,7 @@
 //  $0  Page link
 
 // Throw a MacroDeprecatedError flaw
- mdn.deprecated()
+ mdn.deprecated("The 'Link' macro is deprecated. See https://github.com/orgs/mdn/discussions/187 for details.")
```

Comment:
Should w link to the new Link.deprecated.md page instead ?

Reply:
@teoli2003 I was thinking of doing that, but I was not sure how long that URL will be :) I, therefore, thought of adding a comment to the discussion linking to it. It is also in the context where people are tracking the work to remove said macro.